lms/views/api/groups.py

```diff
@@ -7,9 +7,9 @@
     request_method="GET",
     permission=Permissions.API,
     renderer="json",
-    route_name="blackboard_api.courses.group_sets.list",
+    route_name="api.courses.group_sets.list",
```

Comment:
The diff is finding the blackboard version is now the "generic" one.

They are very similar indeed.